README.md

```
@@ -39,30 +110,521 @@ pnpm run db:migrate
+**Options:**
+- `--env <name>` - Use configuration from `environments.<name>` in `kagaribi.config.ts`
+
+**Examples:**
+```bash
+kagaribi build                    # Build with default configuration
+kagaribi build --env production   # Build with production environment config
+kagaribi build --env staging      # Build with staging environment config
+```
+
+**Output:**
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)